src/MongoDB.Driver.Core/Core/Clusters/MultiServerCluster.cs

```diff
@@ -207,9 +230,9 @@ protected override void RequestHeartbeat()
                     catch (ObjectDisposedException)
                     {
                         // There is a possible race condition here
-                        // due to the fact that we are working 
```

Comment:
Sorry for all the whitespace only diffs. The editor made these changes automatically.